docs/docs/sources/chat-plugin.md

```diff
@@ -20,6 +19,30 @@ connection endpoint](api/http.md#connecting-channels) and setting the `source`
 field in the request payload to `chat_plugin`. You can leave the token parameter
 empty. 
 
+This will give you a `channel_id`, which is required for the following steps.
+
+## Installation
+
+To install the chat plugin UI on your website add the following script tag to your `<head>` section:
+
+```html
+<script>(function (w, d, s, n) {
+        w[n] = w[n] || {};
+        w[n].cid = "CHANNEL_ID";
+        w[n].h = "SCRIPT_HOST";
+        var f = d.getElementsByTagName(s)[0],
+            j = d.createElement(s);
+        j.async = true;
+        j.src = w[n].h + '/s.js';
+        f.parentNode.insertBefore(j, f);
+    })(window, document, 'script', 'airy');
+</script>
+```
+
+You must replace `CHANNEL_ID` with your channel id and `SCRIPT_HOST` with the host of your chat plugin server.
+When using the local vagrant environment `SCRIPT_HOST` should be set to `chatplugin.airy`. 
+
+**Note** `chatplugin.airy` is not publicly accessible by default. So this will only work for local web pages.   
```

Comment:
Perhaps we should discuss this: If we do not use a tunnel, the local version of the chat plugin can only be used locally.

We can also add an example page like in `platform/ui/web/widget/example/index.html` to the container.

Reply:
I second adding the example page (it would also improve the docs!)